pkg/controller/statefulset/stateful_set_control_test.go

```diff
@@ -551,52 +623,6 @@ func UpdateSetStatusFailure(t *testing.T, set *apps.StatefulSet, invariants inva
 	}
 }
 
-func PodRecreateDeleteFailure(t *testing.T, set *apps.StatefulSet, invariants invariantFunc) {
```

Comment:
why do we remove this test? is no longer valid?

Reply:
This test was not removed. There was a code duplication and it was also split into two flavors, please see
- RecreatesFailedPodWithDeleteFailure
- RecreatesSucceededPodWithDeleteFailure